odiglet/pkg/kube/runtime_details/shared.go

```diff
@@ -78,25 +79,30 @@ func runtimeInspection(pods []corev1.Pod, ignoredContainers []string) ([]odigosv
 				log.Logger.V(0).Info("no processes found in pod container", "pod", pod.Name, "container", container.Name, "namespace", pod.Namespace)
 				continue
 			}
-			if len(processes) > 1 {
-				// Currently we don't support multiple processes in the same container, where each one can have a different language
-				// We only take the first process into account, when we'll support multiple processes we'll need to change this.
-				log.Logger.V(0).Info("multiple processes found in pod container, only taking the first one into account", "pod", pod.Name, "container", container.Name, "namespace", pod.Namespace)
+
+			var lang common.ProgrammingLanguage
+			var inspectProc *procdiscovery.Details
+			var detectErr error
+
+			for _, proc := range processes {
+				lang, detectErr = inspectors.DetectLanguage(proc)
+				if detectErr == nil && lang != common.UnknownProgrammingLanguage {
+					inspectProc = &proc
+					break
+				}
+				log.Logger.V(0).Info("error detecting language", "error", detectErr, "process", proc)
```

Comment:
Please remove this logline, which might get called a lot.

Reply:
yes.